src/host.h

```diff
@@ -104,4 +104,6 @@ bool sufmatch (const char **, const char *);
 
 void host_cleanup (void);
 
+void cache_store (const char *host, const char* ip)
```

Comment:
forgotten final `;`

Reply:
Thanks. Also need to rename as name clashes with same name static function in `host.c`.